### PR TITLE
Add --require-hashes to mobile/docs/build.sh and ci/mac_ci_setup.sh

### PR DESCRIPTION
**Pull request by RyanTheOptimist**
Add --require-hashes to mobile/docs/build.sh and ci/mac_ci_setup.sh

Risk Level: Low
Testing: CI
Docs Changes: N/A
Release Notes: N/A
Platform Specific Features: N/A

### CODE REVIEW

**Review by alyssawilk — APPROVED** [review]

File: ci/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#
```
Comment (phlax) [inline ci/requirements.txt:1]: could we move this into `ci/mac` so its obvious why its there - also please commit the `requirements.in`

Comment (RyanTheOptimist) [reply ci/requirements.txt]: Sure! Done!

**Review by phlax — DISMISSED** [review]
lgtm, thanks @RyanTheOptimist 

only small question is whether we need to generate with matching python version - it can affect the deps - almost certainly it wont in this case

**Comment by phlax** [comment]
you need to add an entry in `.github/dependabot.yml`

**Comment by RyanTheOptimist** [comment]
> you need to add an entry in `.github/dependabot.yml`

Ah! Thanks! Done.

**Review by phlax — DISMISSED** [review]

File: ci/mac_ci_setup.sh [hunk 2]
```diff
@@ -64,4 +64,4 @@ fi
 
 bazel version
 
-pip3 install virtualenv
+pip3 install -r mac/requirements.txt --require-hashes
```
Comment (phlax) [inline ci/mac_ci_setup.sh:67]:
```suggestion
pip3 install -r ci/mac/requirements.txt --require-hashes
```

**Review by phlax — APPROVED** [review]

**Review by mattklein123 — APPROVED** [review]

**Pull request merged**